programs/models.py

```diff
@@ -128,6 +128,8 @@ class Program(models.Model):
                                        null=True,
                                        blank=True,
                                        related_name='subplans')
+    created = models.DateTimeField(auto_now_add=True, blank=False, null=False)
```

Comment:
`blank=True` on `auto_now` fields, and I don't think setting it to `False` will override that.